plugins/src/android/OptimoveInitProvider.java

```diff
@@ -10,12 +10,17 @@
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+
 import com.optimove.android.Optimove;
 import com.optimove.android.OptimoveConfig;
+import com.optimove.android.optimobile.OptimoveInApp;
 
 public class OptimoveInitProvider extends ContentProvider {
     private static final String OPTIMOVE_CREDENTIALS = "optimoveCredentials";
     private static final String OPTIMOVE_MOBILE_CREDENTIALS = "optimoveMobileCredentials";
+    private static final String IN_APP_AUTO_ENROLL = "autoEnroll";
```

Comment:
I think these don't match the values in JSON -> XML ?

Reply:
I changed it to auto-enroll